examples/flower_example.rs

```diff
@@ -0,0 +1,35 @@
+use polars::frame::DataFrame;
+use polars::series::Series;
+use polars::error::PolarsResult;
+use polars_core::prelude::*;
+use polars::prelude::NamedFrom;
+use polars::datatypes::Float64Type;
+use matplotlib as plt;
+
+//Yellow Flag Iris, attributed to Iris pseudacorus flowers were measured in Norfolk marshes in 2018 by LCrossman
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let (fig, [[mut ax]]) = plt::subplots()?;
+    let df = DataFrame::default();
+    let pseudacorus_Sepal_Length = Series::new("Pseudacorus_Sepal_Length", &[6.0,6.0,5.8,6.5,5.7,6.8,6.5,6.8,7.0,6.2,6.5,6.6,7.0,7.0,7.5,7.0,6.8,6.5,6.3,6.2,6.9,6.3,6.3,7.0,6.2,6.8,6.2,6.3,5.8,5.9,6.2,6.3,6.9,7.3,7.5,6.4,6.6,6.4,6.7,6.4,6.4,6.6,6.4,6.2,6.3,7.9,6.9,6.2,5.9,6.3]);
+    let pseudacorus_Sepal_width = Series::new("Pseudacorus_Sepal_width", &[4.0,3.1,4.0,3.8,3.4,3.7,4.7,4.0,4.5,3.2,3.9,4.0,4.1,4.0,4.6,4.4,4.0,4.2,3.9,4.0,4.8,4.0,4.2,3.8,3.6,3.3,3.4,3.5,3.1,3.7,4.4,4.5,4.4,4.2,4.2,4.4,4.0,4.2,4.0,4.9,4.3,4.4,3.6,3.8,3.5,4.3,4.6,3.8,3.6,3.8]);
+   let f: PolarsResult<DataFrame> = DataFrame::new(vec![pseudacorus_Sepal_Length,pseudacorus_Sepal_width]);
+   match f {
+       Ok(df) => {   
+          println!("df is {:?}", &df);
+          let x_col = df.column("Pseudacorus_Sepal_Length")?.f64()?;
+          let y_col = df.column("Pseudacorus_Sepal_width")?.f64()?;
+        
+          let x_values: Vec<f64> = x_col.to_vec().iter().map(|&x| x.unwrap()).collect();
```

Comment:
This is not very nice.  I would propose to add [ChunkedArray](https://docs.rs/polars/0.38.1/polars/chunked_array/struct.ChunkedArray.html)<[Float64Type](https://docs.rs/polars/0.38.1/polars/datatypes/struct.Float64Type.html)> to the possible matplotlib data types — under a feature gate so the whole library does not depend on polars.  I can do this but, as I am unfamiliar with polars, a list of data types from this library that one would want to pass to `ax.xy` would be helpful.

Reply:
Hmm I am not that familiar either, it looks like there is a long list in the docs.  At the least some float, int and string types I would say.  I am not sure how it practically pans out with the ChunkedArray<Float64Type> in terms of Int and String